baitap-19-10-2020/bai-1.js

```diff
@@ -0,0 +1,17 @@
+const salaryCalculation = (salary) => {
+    if(salary >= 15000000)
+    {
+        console.log("Thuế thu nhập: ",salary * 30 / 100)
+        console.log("Lương ròng: ",salary * 70 / 100)
+    }
+    else if (salary >= 7000000 && salary < 15000000) 
+    {

```

Comment:
Nhìn kỹ cách đặt dấu `{` và `}`
```suggestion
    } else if (salary >= 7000000) {

```
Bên trên đã kiểm tra `salary >= 15m` rồi, thì làm gì có chuyện xuống dưới này `salary >= 15m` nữa mà phải kiểm tra `salary < 15000000` Tương tự các phần bên dưới